packages/db/tests/models/test-file-model.js

```diff
@@ -105,3 +105,55 @@ test('FilePgModel.upsert() overwrites a file record', async (t) => {
     updatedFile
   );
 });
+
+test('FilePgModel.upsertById() updates the unique indentifiers for a file record', async (t) => {
+  const {
+    knex,
+    filePgModel,
+    granuleCumulusId,
+  } = t.context;
+
+  const file = fakeFileRecordFactory({
+    granule_cumulus_id: granuleCumulusId,
+    checksum_value: cryptoRandomString({ length: 3 }),
+  });
+  const [insertedGranuleId] = await filePgModel.create(knex, file);
```

Comment:
```suggestion
  const [fileCumulusId] = await filePgModel.create(knex, file);
```

Reply:
It's the cumulus ID for a file, not a granule

Reply:
OBE : c8d4942bd